pkg/schedule/windowed_schedule_test.go

```diff
@@ -50,26 +94,28 @@ func TestWindowedSchedule(t *testing.T) {
 					time.Sleep(w.Interval * 2)
 				}
 			}
-			// we should have either 16 or 17 minus 3 missed
-			So(len(r), ShouldBeBetweenOrEqual, 15, 17)
+			// we should have either 17 or 19 minus 4 missed
```

Comment:
For the test with 4 missed responses,  the range has been changed to 15 -18.
Notice that for a test without missed responses, 19- 22 responses are expected to receive.